util/cli/queue.go

```diff
@@ -0,0 +1,123 @@
+package cli
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/src-d/lookout"
+	queue_util "github.com/src-d/lookout/queue"
+	"github.com/src-d/lookout/util/ctxlog"
+	"gopkg.in/src-d/go-queue.v1"
+	_ "gopkg.in/src-d/go-queue.v1/amqp"
+	_ "gopkg.in/src-d/go-queue.v1/memory"
+)
+
+// queueWindow defines the maximum number of unacknowledged jobs that the
+// queue iterator will allow to retrieve. A number of 1 will mean that each
+// job is processed sequentially
+// TODO(carlosms): because the eventHandler is called synchronously, a value
+// greater than 1 will not have any effect for now. We will not call iter.Next()
+// until the previous job is acknowledged or rejected
+const queueWindow = 1
+
+// QueueOptions contains common flags for commands using a Queue
+type QueueOptions struct {
+	Queue  string `long:"queue" env:"LOOKOUT_QUEUE" default:"lookout" description:"queue name"`
+	Broker string `long:"broker" env:"LOOKOUT_BROKER" default:"amqp://localhost:5672" description:"broker service URI"`
+
+	q queue.Queue
+}
+
+// InitQueue initializes the queue from the given cli options.
+func (c *QueueOptions) InitQueue() error {
+	b, err := queue.NewBroker(c.Broker)
+	if err != nil {
+		return err
+	}
+
+	c.q, err = b.Queue(c.Queue)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// EventEnqueuer returns an event handler that pushes event to the queue.
+// InitQueue must be called before.
+func (c *QueueOptions) EventEnqueuer(ctx context.Context) lookout.EventHandler {
```

Comment:
I would better put this function and `RunEventDequeuer` in `queue/queue.go`.

What do you think if we keep `cli` package responsible only for handling command line arguments and move the actual logic to proper packages inside lookout?

Reply:
:+1: change done in 73f0f7f